Mage.Sets/src/mage/cards/p/PersonalIncarnation.java

```diff
@@ -0,0 +1,138 @@
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.cards.p;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DiesTriggeredAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.mana.GenericManaCost;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.RedirectionEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.TargetController;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.players.Player;
+import mage.target.TargetPlayer;
+
+/**
+ *
+ * @author MTGfan
+ */
+public class PersonalIncarnation extends CardImpl {
+
+    public PersonalIncarnation(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{W}{W}{W}");
+        
+        this.subtype.add("Avatar");
+        this.subtype.add("Incarnation");
+        this.power = new MageInt(6);
+        this.toughness = new MageInt(6);
+
+        // {0}: The next 1 damage that would be dealt to Personal Incarnation this turn is dealt to its owner instead. Only Personal Incarnation's owner may activate this ability.
+        SimpleActivatedAbility ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new PersonalIncarnationRedirectEffect(Duration.EndOfTurn, 1), new GenericManaCost(0));
+        ability.setMayActivate(TargetController.OWNER);
+        this.addAbility(ability);
+        // When Personal Incarnation dies, its owner loses half his or her life, rounded up.
+        this.addAbility(new DiesTriggeredAbility(new PersonalIncarnationLoseHalfLifeEffect()));
+    }
+
+    public PersonalIncarnation(final PersonalIncarnation card) {
+        super(card);
+    }
+
+    @Override
+    public PersonalIncarnation copy() {
+        return new PersonalIncarnation(this);
+    }
+}
+
+class PersonalIncarnationRedirectEffect extends RedirectionEffect {
+
+    public PersonalIncarnationRedirectEffect(Duration duration, int amountToRedirect) {
+        super(duration, amountToRedirect, true);
+        staticText = "The next 1 damage that would be dealt to {this} this turn is dealt to its owner instead.";
```

Comment:
If you're going to assume a duration and amount in the static text please assume the same in the actual code.